pkg/utils/hash.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package utils
+
+import (
+	"crypto"
+	"fmt"
+	"hash"
+
+	multihash "github.com/multiformats/go-multihash"
+)
+
+// ComputeMultihash will compute the hash for the supplied bytes using multihash code
+func ComputeMultihash(multihashCode uint, bytes []byte) ([]byte, error) {
+
+	if len(bytes) == 0 {
+		return nil, fmt.Errorf("empty bytes")
+	}
+
+	h, err := GetHash(multihashCode)
+	if err != nil {
+		return nil, err
+	}
+
+	if _, hashErr := h.Write(bytes); hashErr != nil {
+		return nil, hashErr
+	}
+	hash := h.Sum(nil)
+
+	return multihash.Encode(hash, uint64(multihashCode))
+
+}
+
+// GetHash will return hash based on specified multihash code
+func GetHash(multihashCode uint) (h hash.Hash, err error) {
+
+	switch multihashCode {
+	case 18:
```

Comment:
A comment on what '18' means would be helpful for someone who is new to code.

Reply:
I agree I will define meaningful constant